Bookish/Bookish.Web/Models/AddBookViewModel.cs

```diff
@@ -34,7 +34,12 @@ private IEnumerable<string> CreateErrorMessage(bool isbnError)
 
             if (isbnError)
             {
-                errorMessage.Add("That ISBN has already been used");
+                errorMessage.Add("ISBN already used");
+            }
+
+            if (Isbn.Replace("-", "").Length != 13 && Isbn != "")
```

Comment:
See https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation, in particular the Validating against a regular expression section, for how to do this in the cshtml 🙂